Content.Shared/Pulling/Systems/SharedPullingSystem.Actions.cs

```diff
@@ -54,6 +55,12 @@ public bool CanPull(EntityUid puller, EntityUid pulled)
                     return false;
                 }
             }
+            // Can't pull people who are cuffed if someone else is pulling them.
+            // You have to stun the puller or something first.
+            // Throwing sec a bone.
+            if (TryComp<SharedPullableComponent>(pulled, out var pullable) && pullable.Puller != null // Are they being pulled by someone else?
+            && TryComp<SharedCuffableComponent>(pulled, out var cuffable) && !cuffable.CanStillInteract) // Are they cuffed?
+                return false;
 
             var startPull = new StartPullAttemptEvent(puller, pulled);
             RaiseLocalEvent(puller, startPull);
```

Comment:
Using the event would be better.

Reply:
Sure, subscribed to on cuffable okay?

Reply:
yeah though the event is raised on puller so might need to raise it again on the pulled.

Reply:
The event is raised on the puller and we're checking things on the pulled.

Reply:
raising it again on pulled would be bad (at least with the same event) then you wouldnt be able to pull ppl that are stunned and what not i think?

Reply:
could just make new event for the pulled